vms/proposervm/block.go

```diff
@@ -82,9 +82,6 @@ func (p *postForkCommonComponents) Verify(parentTimestamp time.Time, parentPChai
 	}
 
 	childPChainHeight := child.PChainHeight()
-	if childPChainHeight < parentPChainHeight {
-		return errPChainHeightNotMonotonic
-	}
```

Comment:
Why is this removed?

Reply:
The p-chain height doesn't change anymore. So this check is no longer required.

Reply:
It should still not be removed. We talked about this. We are not touching P-Chain related logic for now.